src/templates/hl7v2/OML_O21.hbs

```diff
@@ -3,83 +3,107 @@
     "type": "transaction",
     "entry": [
         {{#with (getFirstSegments msg.v2 'PID' 'PD1' 'PV1' 'PV2' 'ORC')}}
-            
-            {{assert ORC "Missing ORC"}},
```

Comment:
Do I understand it correctly that instead of "assert" (check and throw exceptions if the field does not exist), we wrap an if condition outside it when the field is used?

Reply:
Yes but partially, the "if" statements are used mainly to make sure there are no empty "ID" field(4cfe8d6d-3fc8-3e41-b921-f204be18db31), as well as doing part of what "assert" statement did.

The main function of "assert" statement will be replaced by a preprocessor(basically a V2 message validator) to execute more comprehensive restrictions.

Reply:
Got it. Thanks!